scip.proto

```diff
@@ -240,25 +244,27 @@ enum SyntaxKind {
   IdentifierConstant = 9;
   // `var X = "hello"` in Go
   IdentifierMutableGlobal = 10;
-  // both parameter definition and references
+  // Parameter definition and references
   IdentifierParameter = 11;
-  // identifiers for variable definitions and references within a local scope
+  // Identifiers for variable definitions and references within a local scope
   IdentifierLocal = 12;
-  // Used when identifier shadowes some other identifier within the scope
+  // Identifiers that shadow other identifiers in an outer scope
   IdentifierShadowed = 13;
-  // `package main`
+  // Identifier representing a unit of code abstraction:
+  // `os` in `import os` in Python, `main` in `package main` in Go.
   IdentifierModule = 14;
 
-  // Function call/reference
+  // Function references, including calls
   IdentifierFunction = 15;
   // Function definition only
   IdentifierFunctionDefinition = 16;
 
-  // Macro call/reference
+  // Macro references, including invocations
   IdentifierMacro = 17;
   // Macro definition only
   IdentifierMacroDefinition = 18;
 
+  // TODO[Varun]: What does 'including namespaces' mean?
```

Comment:
TODO for clarification.

Reply:
I'm guessing it's references to package identifiers, for example `java` and `io` in `java.io.File`. However, we should have a dedicated syntax kind for identifiers to packages/namespaces. IntelliJ gives them different colors than type identifiers and I find it very helpful to distinguish between them when reading code.

Reply:
We have `IdentifierModule`; that seems applicable here.

```
  // Identifier representing a unit of code abstraction:
  // `os` in `import os` in Python, `main` in `package main` in Go.
  IdentifierModule = 14;
```

It seems like we should reuse that for packages and namespaces, and remove the point about namespaces from this comment?

Reply:
I missed that one. It's a bit confusing how we currently mix package/module/namespace in the spec. For example, we use `<package-name>` to refer to an npm package name, while `Descriptor.Suffix.Package` and `SyntaxKind.IdentifierModule` both refer to a C++ namespace, python/js module, or Java package.

Reply:
It is BTW backwards compatible to rename `Suffix.Package` and `IdentifierModule` into something else. The term "namespace" might be the most self-explanatory name.

Reply:
We're consistently using namespace now, so this is resolved.